docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -1,122 +1,44 @@
-
-```
-kubectl get nodes
-NAME                              STATUS   ROLES           AGE   VERSION
-mgmt-clrt4                        Ready    control-plane   3d22h   v1.27.1-eks-61789d8
-mgmt-md-0-5557f7c7bxsjkdg-l2kpt   Ready    <none>          3d22h   v1.27.1-eks-61789d8
-```
-
-**Verify an EKS Anywhere cluster from the management cluster**
-
-Set up `CLUSTER_NAME` and `KUBECONFIG` environment variable for the management cluster:
 ```
-export CLUSTER_NAME=mgmt
-export KUBECONFIG=${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
+kubectl get nodes | grep -i control-plane
+NAME                                 STATUS   ROLES           AGE   VERSION
+eksapoc-lkfnr                        Ready    control-plane   10d   v1.27.1-eks-75a5dcc
+eksapoc-wjq2c                        Ready    control-plane   10d   v1.27.1-eks-75a5dcc
+eksapoc-zqsv9                        Ready    control-plane   10d   v1.27.1-eks-75a5dcc
 ```
 
-You may verify the control plane is up and running by filtering the pods by the `control-plane=controller-manager` label.
-```
-kubectl get pod -A -l control-plane=controller-manager
-NAMESPACE                           NAME                                                             READY   STATUS    RESTARTS      AGE
-capi-kubeadm-bootstrap-system       capi-kubeadm-bootstrap-controller-manager-8665b88c65-v982t       1/1     Running   0             3d21h
-capi-kubeadm-control-plane-system   capi-kubeadm-control-plane-controller-manager-67595c55d8-z7627   1/1     Running   0             3d21h
-capi-system                         capi-controller-manager-88bdd56b4-wnk66                          1/1     Running   0             3d21h
-capv-system                         capv-controller-manager-644d9864dc-hbrcz                         1/1     Running   1 (15h ago)   3d21h
-eksa-packages                       eks-anywhere-packages-784c6fc8b9-2t5nr                           1/1     Running   0             3d21h
-etcdadm-bootstrap-provider-system   etcdadm-bootstrap-provider-controller-manager-6bcdd4f5d7-wvqw8   1/1     Running   0             3d21h
-etcdadm-controller-system           etcdadm-controller-controller-manager-6f96f5d594-kqnfw           1/1     Running   0             3d21h
-```
-
-You may also check the status of the cluster control plane resource directly. 
-This can be especially useful to verify clusters with multiple control plane nodes after an upgrade.
-
-```
-kubectl get kubeadmcontrolplanes.controlplane.cluster.x-k8s.io -n eksa-system
-NAME   CLUSTER   INITIALIZED   API SERVER AVAILABLE   REPLICAS   READY   UPDATED   UNAVAILABLE   AGE     VERSION
-mgmt   mgmt      true          true                   1          1       1                       3d22h   v1.27.1-eks-1-27-4
-w01    w01       true          true                   1          1       1         0             16h     v1.27.1-eks-1-27-4
-```
-
-You may check the status of the cluster resource directly.
```

Comment:
We cannot remove this. This is a key troubleshooting technique to observe specific parts of the system